staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -578,6 +578,8 @@ type VirtualMachineInstanceNetworkInterface struct {
 	InfoSource string `json:"infoSource,omitempty"`
 	// Specifies how many queues are allocated by MultiQueue
 	QueueCount int32 `json:"queueCount,omitempty"`
+	// PodConfigDone specifies if the corresponding pod interface is properly configured by CNI
```

Comment:
Is `by CNI` accurate?
We are not expressing here that it is not supported by the pod network/primary and the fact it is only multus related.

We could raise this flag when detecting the pod-nic, but I think we do not go that far. Should we do that for consistency?

Reply:
? The default primary network will also have this attribute enabled. 

I don't want to detect anything on the pod: I want to rely on the neat API I have that reports the CNI result translated into the cncf network-status.

Reply:
I think we have an interest not to be locked on a specific implementation here.
But this is not the point really, we need to be accurate with the parameter description.. if we are not intending to expose this for non-multus pod nics, then we somehow need to express this in the description.
I found it challenging to describe, that is why I thought maybe we can extend it for the primary one.

Not really sure why you see this as problematic, but I do not really mind with just trying to make the description more accurate then.

Reply:
> I think we have an interest not to be locked on a specific implementation here. But this is not the point really, we need to be accurate with the parameter description.. if we are not intending to expose this for non-multus pod nics, then we somehow need to express this in the description. I found it challenging to describe, that is why I thought maybe we can extend it for the primary one.

Again, the pod primary interfaces are also subject to this flag. The only thing multus plays in this part is that it is the component translating CNI result to the network status - for **all** interfaces. 

> 
> Not really sure why you see this as problematic, but I do not really mind with just trying to make the description more accurate then.

OK, I guess fair enough. IMO the description is accurate. In yours, it clearly isn't; what do you suggest ?

Reply:
> Again, the pod's primary interface are also subject to this flag. The only thing multus plays in this part is that it is the component translating CNI result to the network status - for all interfaces.

I guess this is the part I did not understand well.
If Multus is installed, it will also report the primary network because it acts as an intermediator.
If this is correct, I think this clarifies for me the thing and the description is indeed clear.

Reply:
Yes, that's exactly it. Resolving the comment.